main.gms

```diff
@@ -1143,6 +1143,19 @@ parameter
 ;
   c_changeProdCost = 5;  !! def = 5
 *'
+parameter
+  cm_LearningSpillover      "Activate Learningspillover from foreign capacity in learning technogolies"
+;
+  cm_LearningSpillover = 1; !! def 1 = Learningspillover activated (set to 0 to deactivate)
+*'
+*' * if Learningspillover is deactivated, foreign capacity is set to the level of 2020 in technology learning.
+*' * In combination with endogenous carbon pricing (e.g., in NDC), the deactivated Learningspillover will lead to higher overall carbon prices. Can be solved by setting carbonprice to exogenous (config).
```

Comment:
sorry that I have a tiny other point - could you change the order of the explanations?

if you have a sentence starting "This means that ..." after "In combination with endogenous ...", one could misunderstand the "this" to refer to the previous sentence, and not the "if Learningspillover is deactivated ..."
so I would put the "In combination with endogenous" at the end of the explanation :-)

Reply:
Yes, I see your point and have changed the order (commit 9)!